convention-plugins/convention/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    `kotlin-dsl`
+    `maven-publish`
+}
+
+
+group = "com.michiganlabs.hoverboard.convention-plugins"
+version = "1.0"
+
+//publishing {
```

Comment:
Can delete this for now, but it'd be interesting to see if hoverboard could create common plugins for consumer projects as well.